pom.xml

```diff
@@ -615,16 +615,17 @@
             <groupId>org.jvnet.staxex</groupId>
             <artifactId>stax-ex</artifactId>
           </exclusion>
-          <!--
-            SPARK-27611: Exclude redundant javax.activation implementation, which
-            conflicts with the existing javax.activation:activation:1.1.1 dependency.
-            -->
           <exclusion>
-            <groupId>jakarta.activation</groupId>
```

Comment:
In the current `jaxb-runtime` version, we need `jakarta.activation`, otherwise there will be `java.lang.NoClassDefFoundError: jakarta/activation/DataSource
` errors, both when exporting PMML models and when using jersey.

Reply:
some exclusions are invalid now, please remove that.

Reply:
Hmm, I found that the `jakarta.xml.bind:jakarta.xml.bind-api` exclusion in `jersey-server` can be removed, but the `com.sun.activation:jakarta.activation` exclusion in `jersey-common` can still be retained, because this dependence is currently useless.

Reply:
I mean `com.sun.xml.fastinfoset:FastInfoset` and `org.jvnet.staxex:stax-ex`, they are marked as optional in `org.glassfish.jaxb:jaxb-runtime:4.0.5`

Reply:
Thanks for pointing this out, I updated it